src/index.ts

```diff
@@ -38,16 +38,19 @@
 
 import * as arrify from 'arrify';
 import {replaceProjectIdToken} from '@google-cloud/projectify';
+import {Operation as GaxOperation} from 'google-gax/build/src/longrunning';
```

Comment:
Do you need to reach into the full path here, or is this type exported from the index of `google-gax`?